### PR TITLE
Feat - Added login and signup pages

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
### Change Log

- Added Login Page
- Added Signup Page

login
signup

### CODE REVIEW

File: css/utils/vars.css [hunk 1]
```diff
@@ -2,4 +2,5 @@
   --primary--color: #6d4930;
   --text--primary--color: white;
   --nav--input--border: black;
+  --centered--box-color: white;
```
Comment (NeerajVodala) [inline css/utils/vars.css]: recommended to use hex or rgba values(cause different browsers use different shades for same color)

**Review by NeerajVodala — APPROVED** [review]
Everything looks fine to me (added one comment, please check).
All the best.

**Review by ark2002 — COMMENTED** [review]
other from Neeraj's recommendation everything seem fine

**Comment by SRUJANA-PENUGONDA13** [comment]
Thank You @ark2002 @NeerajVodala for reviewing

**Pull request merged**